### PR TITLE
Feature/teams page

### PR DESCRIPTION
**Pull request by ShadySloth**
Implementation of the teams page and modal for creation of new teams.

### CODE REVIEW

File: src/FE/src/app/add-profile-dialog/add-profile-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      if (value == 1) {
+        this.profileForm.get('annual_hours')?.disable();
+      } else {
+        this.profileForm.get('annual_hours')?.enable();
+      }
+    });
+  }
+
+  addProfile() {
+    //todo: add profile to db and update table
```
Comment (DanKjaer) [inline src/FE/src/app/add-profile-dialog/add-profile-dialog.component.ts:69]: todo?

File: src/FE/src/app/add-teams-dialog/add-teams-dialog.component.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<h2 mat-dialog-title>{{ 'ADD_NEW_TEAM' | translate }}</h2>
+<mat-dialog-content class="container" [formGroup]="teamForm">
+  <!-- name input -->
+  <mat-form-field>
+    <mat-label>
+      {{ 'NAME' | translate }}
+    </mat-label>
+    <input matInput placeholder="Team Easy On" formControlName="name">
```
Comment (DanKjaer) [inline src/FE/src/app/add-teams-dialog/add-teams-dialog.component.html:8]: https://www.youtube.com/watch?v=BIkx9WpOt3A

**Review by DanKjaer — CHANGES_REQUESTED** [review]
Ikke helt holdt dig indenfor scoped af branchen - ellers ok. 2 kommentarer

File: src/FE/src/app/add-profile-dialog/add-profile-dialog.component.ts (hunk 1 above)
Comment (ShadySloth) [reply src/FE/src/app/add-profile-dialog/add-profile-dialog.component.ts:69]: det betyder at det skal laves senere Dan.

Comment (hoeeg79) [reply src/FE/src/app/add-profile-dialog/add-profile-dialog.component.ts:69]: man kunne argumentere at hele denne component slet ikke burde findes i denne pr

**Review by hoeeg79 — APPROVED** [review]

**Pull request merged**